src/Components/WebAssembly/WebAssembly/src/Services/HttpClientServiceCollectionExtensions.cs

```diff
@@ -1,31 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Net.Http;
-using Microsoft.AspNetCore.Components;
-
-namespace Microsoft.Extensions.DependencyInjection
-{
-    public static class HttpClientServiceCollectionExtensions
-    {
-        /// <summary>
-        /// Adds a <see cref="HttpClient" /> instance to the <paramref name="serviceCollection" /> that is
-        /// configured to use the application's base address (<seealso cref="NavigationManager.BaseUri" />).
-        /// </summary>
-        /// <param name="serviceCollection">The <see cref="IServiceCollection" />.</param>
-        /// <returns>The configured <see cref="IServiceCollection" />.</returns>
-        public static IServiceCollection AddBaseAddressHttpClient(this IServiceCollection serviceCollection)
```

Comment:
Why remove this API? It's used in the default project template BTW.

Reply:
Oh I see, it comes from @pranavkm's comment at https://github.com/dotnet/aspnetcore/issues/19910#issue-582660183

I'm totally open to removing this API, as it doesn't make the project template much more complicated to add the `HttpClient` manually. The project template would need to be updated as part of this PR though: https://github.com/dotnet/aspnetcore/blob/blazor-wasm-preview4/src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Program.cs#L21

Reply:
Fixed.